drm/drm_lock.c

```diff
@@ -44,6 +44,33 @@ __FBSDID("$FreeBSD$");
 #include "drm_legacy.h"
 #include "drm_internal.h"
 
+#ifdef __aarch64__
+#undef cmpxchg
+#define	cmpxchg(ptr, old, new) ({				\
```

Comment:
This should be fixed in `linuxkpi`, arm64 does not have `atomic_fcmpset_*` for 8 and 16 bit variables, but they were referenced in this macro